test/e2e/e2e.go

```diff
@@ -0,0 +1,153 @@
+/*
+Copyright 2015 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"testing"
+	"time"
+
+	kutil_pf "github.com/appscode/kutil/tools/portforward"
+	"github.com/golang/glog"
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/config"
+	"github.com/onsi/ginkgo/reporters"
+	"github.com/onsi/gomega"
+	apiext "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	apiext_clientset "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	runtimeutils "k8s.io/apimachinery/pkg/util/runtime"
+	core "k8s.io/client-go/kubernetes/typed/core/v1"
+
+	api "github.com/presslabs/mysql-operator/pkg/apis/mysql/v1alpha1"
+	"github.com/presslabs/mysql-operator/pkg/util/kube"
+	"github.com/presslabs/mysql-operator/test/e2e/framework"
+	"github.com/presslabs/mysql-operator/test/e2e/framework/ginkgowrapper"
+
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+)
+
+const (
+	operatorNamespace = "mysql-operator"
+	releaseName       = "operator"
+
+	orchestratorPort = 3000
+)
+
+var orcTunnel *kutil_pf.Tunnel
+
+var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
+	// ginkgo node 1
+	ginkgo.By("Install operator")
+	HelmInstallChart(releaseName)
```

Comment:
Maybe it's better to move `helm.go` to framework.

Reply:
Way? I need it in `BeforeSuite` and is not used in framework package.